extensions/smallrye-openapi/deployment/src/test/java/io/quarkus/smallrye/openapi/test/jaxrs/OpenApiResourceSecuredAtMethodLevel.java

```diff
@@ -22,6 +24,10 @@ public String secureEndpointWithoutSecurityAnnotation() {
         return "secret";
     }
 
+    @APIResponses({
+            @APIResponse(responseCode = "401", description = "Who are you?"),
+            @APIResponse(responseCode = "403", description = "You cannot do that.")
+    })
```

Comment:
Rather add a new test method so we still have a test that test the auto creation of APIResponse in the case it's not provided and another that test the case that do not create them (like you have now)